dstep/translator/Translator.d

```diff
@@ -217,7 +217,10 @@ class Translator
 
     void translateVarDecl(Output output, Cursor cursor, Cursor parent)
     {
-        variable(output, cursor, "extern __gshared ");
+        version (D1)
+            variable(output, cursor, "extern ");
+        else
+            variable(output, cursor, "extern __gshared ");
```

Comment:
Please put the string in a local variable and version that instead of the whole call.